iterative/aws/provider.go

```diff
@@ -486,17 +486,19 @@ func GetAvailabilityZone(region string) string {
 }
 
 func getInstanceType(instanceType string, instanceGPU string) string {
-	instanceTypes := make(map[string]string)
-	instanceTypes["m"] = "m5.2xlarge"
-	instanceTypes["l"] = "m5.8xlarge"
-	instanceTypes["xl"] = "m5.16xlarge"
-	instanceTypes["m+k80"] = "p2.xlarge"
-	instanceTypes["l+k80"] = "p2.8xlarge"
-	instanceTypes["xl+k80"] = "p2.16xlarge"
-	instanceTypes["m+v100"] = "p3.xlarge"
-	instanceTypes["l+v100"] = "p3.8xlarge"
-	instanceTypes["xl+v100"] = "p3.16xlarge"
-
+	instanceTypes := map[string]string{
```

Comment:
This will be allocated on every run of this function. Granted, the penalty is small, but I'd probably define this map at the package level.

Reply:
I had several PRs ago suggested these all be unified and a const but was overruled. (previously this was about regions, but I think the point stands: https://github.com/iterative/terraform-provider-iterative/pull/344)

So again I agree but I think we should resolve this as a separate tech debt PR/discussion.

Reply:
https://github.com/iterative/terraform-provider-iterative/issues/720